storybook/src/components/logo/logo.css-template.ts

```diff
@@ -0,0 +1,21 @@
+import { Logo } from "dso-toolkit";
+
+import { html, nothing } from "lit-html";
+import { classMap } from "lit-html/directives/class-map.js";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+import { unsafeHTML } from "lit-html/directives/unsafe-html.js";
+import { ComponentImplementation } from "../../templates";
+
+export const cssLogo: ComponentImplementation<Logo> = {
+  component: "Logo",
+  implementation: "html-css",
```

Comment:
Dit component gaat geen HTML/CSS implementatie krijgen. Je mag je focussen op het Web Component. Er hoeven dus ook geen Logo mixins te komen, we hoeven immers geen styling te centraliseren.